internal/wrappers/results-modifier.go

```diff
@@ -80,7 +80,13 @@ func (s *ScanResultNode) UnmarshalJSON(data []byte) error {
 }
 
 func uintValue(value int, name, filename string) uint {
-	var r uint = 0
+	var r uint
+	// Length must be bigger than 0
+	if name == length {
```

Comment:
Might be picky but comparing a special case for length seems strange. What if we pass a default value as an argument to the function and do "var r uint = defaultValue"?